requirements.txt

```diff
@@ -0,0 +1,4 @@
+python-dotenv
+psycopg
+numpy
+fire
```

Comment:
Seems dangerous not to specify versions for these; virtually guarantees that one of these projects will release a breaking change that breaks `pip install`.

Would recommend using `poetry` here instead of `pip`—its lockfile support is way way better than `pip`'s.